apps/web/frontend/components/timeSheetView/workHoursSheet.tsx

```diff
@@ -31,10 +32,10 @@ export const WorkHoursSheet = (): JSX.Element => {
     <div className="flex flex-col">
       <section className="mt-10 grid w-full grid-cols-3 gap-2 text-left">
         <article className="contents border-y text-lg">
-          <strong>Project</strong>
-          <strong>Task</strong>
-          <strong>Person</strong>
-          <strong>Hours</strong>
+          <PageHeading>Project</PageHeading>
```

Comment:
this is not PageHeading.

add PageHeading to the pages/sheet.page.tsx and not here.